src/marqo/tensor_search/tensor_search.py

```diff
@@ -1325,18 +1324,30 @@ def _lexical_search(
     return {'hits': res_list}
 
 
-def construct_vector_input_batches(query: Union[str, Dict], index_info) -> Union[List[str], List[List[str]]]:
+def construct_vector_input_batches(query: Union[str, Dict], index_info) -> Tuple[List[str], List[str]]:
```

Comment:
Perhaps this could be better as `Dict[ModalityType, List[str]]`. It'd be easier to read, and if we add other modalities more simply

Reply:
Yep that would make sense. But it might be a while before we add another modality, and we could cross that bridge when we get to it.